topsail/_common.py

```diff
@@ -162,6 +162,21 @@ def _run(self):
         print(f"Using '{env['ANSIBLE_CACHE_PLUGIN_CONNECTION']}' to store ansible facts.")
         Path(env["ANSIBLE_CACHE_PLUGIN_CONNECTION"]).parent.mkdir(parents=True, exist_ok=True)
 
+        # We configure the roles path dinamycally appending them to the defaults
+        top_dir_roles = os.path.join(top_dir, 'roles')
+        top_dir_roles_list = [os.path.join(top_dir_roles, entry) for entry in os.listdir(top_dir_roles) if os.path.isdir(os.path.join(top_dir_roles, entry))]
```

Comment:
Can you use `pathlib`?
```
top_dir_roles = top_dir / "roles"
```
and maybe:
```
top_dir_roles_list = top_dir_roles.glob("*")
```
(can be filtered with`os.path.isdir`)

Reply:
sure thing pathlib is modern stuff